arch/risc-v/src/mpfs/mpfs_ihc.c

```diff
@@ -1256,8 +1284,12 @@ static void mpfs_rptun_worker(void *arg)
       return;
     }
 
-  DEBUGASSERT((g_vq_idx - VRING0_NOTIFYID) < VRINGS);
-  info = &g_mpfs_virtqueue_table[g_vq_idx - VRING0_NOTIFYID];
+  if (g_vq_idx >= VRINGS)
```

Comment:
This is safe as it is. I'm just wondering, does it no longer support 2 channels / `CONFIG_MPFS_IHC_RPMSG_CH2` (`VRINGS` = 2, `VRING0_NOTIFYID` = 2 or `VRING1_NOTIFYID` = 3) ? Not even sure we're ever going to need it, just out of curiosity.